index.html

```diff
@@ -0,0 +1,1928 @@
+                />
+                <div
+                  class="py-2 position-absolute pl-0"
+                  style="background: white; bottom: 0; width: 100%;"
+                >
+                  <span class="cover py-1 px-2">COVER</span>
+                  <span class="biz py-1 px-2">BUSINESS</span>
+                </div>
+              </div>
+              <h3 class="section-minititle">
+                The Coronavirus Will Change How We Work Forever
+              </h3>
+            </div>
+          </div>
+          <div class="col-sm">
+            <div class="feature1">
+              <div class="position-relative">
+                <img
+                  src="https://d.newsweek.com/en/full/1576924/cul-ps-reedus.jpg"
+                  alt="Wuhan"
+                  style="height: 311px;"
+                  class=""
+                />
+                <div
+                  class="py-2 position-absolute pl-0"
+                  style="background: white; bottom: 0; width: 100%;"
+                >
+                  <span class="cover py-1 px-2">COVER</span>
+                  <span class="biz py-1 px-2">BUSINESS</span>
+                </div>
+              </div>
+              <h3 class="section-minititle">
+                The Coronavirus Will Change How We Work Forever
+              </h3>
+            </div>
+          </div>
+          <div class="col-sm">
+            <div class="feature1">
+              <div class="position-relative">
+                <img
+                  src="https://d.newsweek.com/en/full/1578027/emotional-postponement.webp"
+                  alt="Wuhan"
+                  style="height: 311px;"
+                  class=""
+                />
+                <div
+                  class="py-2 position-absolute pl-0"
+                  style="background: white; bottom: 0; width: 100%;"
+                >
+                  <span class="cover py-1 px-2">COVER</span>
+                  <span class="biz py-1 px-2">BUSINESS</span>
+                </div>
+              </div>
+              <h3 class="section-minititle">
+                The Coronavirus Will Change How We Work Forever
+              </h3>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="editor mx-1 px-3 py-3" style="background: white;">
+        <div
+          class="d-flex justify-content-center align-items-center pb-2 border-bottom border-dark"
+          style="
+            font-family: Roboto Condensed, sans-serif;
+            font-size: 22px;
+            font-weight: 600;
+          "
+        >
+          <span class="iconn"></span>
+          EDITOR'S PICK
+        </div>
+        <div class="row pt-3">
+          <div class="col-sm">
+            <div class="feature1">
+              <div class="position-relative">
+                <img
+                  src="https://d.newsweek.com/en/full/1580936/huawei.jpg"
+                  alt="Wuhan"
```

Comment:
The images in all the categories' sections don't have the right sizing. Please take a look at the original page and adjust this using bootstrap classes.

Also, check the width of each column. Your page has a lot of space between the articles and the three articles on the right side of the sections are wider than they should. Please adjust them using bootstrap classes.

On the mobile version. the three articles on the right should appear below the two articles. Yours don't. Check the image proportions on the mobile version too.
![Screenshot from 2020-04-13 13-09-29](https://user-images.githubusercontent.com/51926937/79146543-2f407880-7d88-11ea-871b-ec688abbb0a5.png)